float64 err_x
float64 err_y
float64 err_z
float64 vel_x_ctrl
float64 vel_y_ctrl
float64 w_ctrl
float64 vel_lin_calcolata
float64 vel_ang_calcolata
float64 yaw_control


